homeassistant/components/airly/__init__.py

```diff
@@ -150,7 +150,7 @@ def __init__(
         """Initialize."""
         self.latitude = latitude
         self.longitude = longitude
-        self.airly = Airly(api_key, session)
+        self.airly = Airly(api_key, session, language=hass.config.language)
```

Comment:
Their documentation seems to suggest only English and Polish are supported, https://developer.airly.org/en/docs#general.language  Did you test to confirm it still works if a language other than EN or PL is specified?

Reply:
Yes, of course, if you use a language other than the supported in the request, the API will return English strings. If in the future they add support for new languages (there are many Airly stations in Germany so German is the most likely candidate), they will just start working in integration.